WordPress/wordpress_startup_script.md

```diff
@@ -0,0 +1,36 @@
+# Startup Script in WordPress
```

Comment:
The file for startup script is already present. Either delete that or modify it with your new content. 
https://github.com/Azure/wordpress-linux-appservice/blob/main/WordPress/running_post_startup_scripts.md 

Images are also present in the media folder
https://github.com/Azure/wordpress-linux-appservice/blob/main/WordPress/media/post_startup_script_1.png
...